.github/workflows/build-pullrequest.yml

```diff
@@ -28,7 +28,20 @@ jobs:
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
-
+      - name: Apply formatter
+        run: ./gradlew spotlessApply
+      - name: Commit formatting changes
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git add .
+          git commit -m "Apply formatter" -a
+        continue-on-error: true
+      - name: Push formatting changes
+        uses: ad-m/github-push-action@master
```

Comment:
Can you please pin a specific commit instead of blindly trusting master? If the action would be compromised so would our repository.